x-pack/plugins/observability_solution/apm/server/routes/assets/services/get_services_from_assets.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { termQuery, rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { ASSET_TYPE, FIRST_SEEN, LAST_SEEN } from '../../../../common/es_fields/assets';
+import { AssetsESClient } from '../../../lib/helpers/create_es_client/create_assets_es_client/create_assets_es_clients';
+import { withApmSpan } from '../../../utils/with_apm_span';
+
+export const MAX_NUMBER_OF_SERVICES = 1_000;
```

Comment:
Limit the number to 1_000 

1. Be consistent with services from apm signal 
2. while I don't expect a performance issue fetching all services from 'assets' index,  I foresee a possible performance issue with the sorting mechanism (either Alerts or any other column)